### PR TITLE
Display extended info about parking charge zones

### PR DESCRIPTION
**Pull request by juhomakkonen**
# Display extended info about parking charge zones
### Cards 175 and partly 173.

-----------------------------------------------------------------------------------------------
#### Display extended info about parking charge zones
 1. src/views/MobilitySettingsView/components/ExtendedInfo/ExtendedInfo.js
     * New component that will display extended info about parking charge zones.
   
 2. src/views/MobilitySettingsView/components/ExtendedInfo/index.js
     * Import, then export from index -file.
    
 3. src/views/MobilitySettingsView/components/ExtendedInfo/styles.js
     * Component styles.
     
 4. src/views/MobilitySettingsView/MobilitySettingsView.js
     * Import ExtendedInfo and create translations object.
   
 5. src/i18n/en.js & src/i18n/fi.js & src/i18n/sv.js
     * Add new texts/translations.
    
 6. browserTests/browserTest.js
      * Comment out test that tested transit/bus stops since we don't display those on the map.

 6. browserTests/unitPageTest.js
      * Change unit id.

### CODE REVIEW

File: browserTests/browserTest.js [hunk 1]
```diff
@@ -61,13 +61,13 @@ test('Contrast does change', async (t) => {
   ;
 });
 
-fixture`Map tests`
+/* fixture`Map tests`
```
Comment (aceViilee) [inline browserTests/browserTest.js:64]: Test not working so commented out?

**Review by aceViilee — COMMENTED** [review]
LGTM. Some comments though to tests

File: browserTests/browserTest.js (hunk 1 above)
Comment (juhomakkonen) [reply browserTests/browserTest.js:64]: That tests if bus stops are visible on the map and we don't display those yet.

Comment (aceViilee) [reply browserTests/browserTest.js:64]: Cool

**Review by aceViilee — APPROVED** [review]
LGTM

**Pull request merged**